test/core/tsi/alts/zero_copy_frame_protector/alts_zero_copy_grpc_protector_test.cc

```diff
@@ -37,6 +37,8 @@ constexpr size_t kSmallBufferSize = 16;
 constexpr size_t kLargeBufferSize = 16384;
 constexpr size_t kChannelMaxSize = 2048;
 constexpr size_t kChannelMinSize = 128;
+constexpr size_t kOverhead = 24;
```

Comment:
nit: Can we call this variable kFrameOverhead and clarify in a comment that it takes into account both the 8-byte header and the 16-byte authentication tag?